tf2_msgs/package.xml

```diff
@@ -10,17 +11,18 @@
   <license>BSD</license>
 
   <url type="website">http://www.ros.org/wiki/tf2_msgs</url>
-    
-  <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>actionlib_msgs</build_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>message_generation</build_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
-  <run_depend>actionlib_msgs</run_depend>
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>message_generation</run_depend>
+  <!-- <depend>actionlib_msgs</depend> -->
+  <depend>builtin_interfaces</depend>
+  <depend>geometry_msgs</depend>
 
-</package>
+  <exec_depend>rosidl_default_runtime</exec_depend>
```

Comment:
There is not such package. I just was exploring that for ros2/rclcpp#181. 

I copied this from common_interfaces and rcl_interfaces. Should we be removing it everywhere? @wjwwood mentioned it might be necessary for packaging but if we don't have a need for it now we should probably drop it everywhere.